### DISCUSSION TITLE
salt-call state.highstate error

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#7843] [Unsupported Versions] [answered]
Hello!

When I launch `salt-call state.highstate` on my Manager node I get always the following errrors:

![image](https://user-images.githubusercontent.com/81571847/164640516-43c72db9-0b69-4676-b83c-daca9cf763aa.png)

However, it seems to work anyway. Do I have to worry about these errors?

![image](https://user-images.githubusercontent.com/81571847/164642384-e193139b-ac2a-4a21-a070-faf471d7a5c1.png)

Cheers, Ben

**Comment by dougburks** [comment, marked as answer]
What is the output of the following?
```
cat /etc/soversion /etc/sohotfix
```

**Reply by ben-sec** [reply]
2.3.110

**Reply by dougburks** [reply]
Did you run that command on your manager? Other nodes won't have `/etc/sohotfix`.

**Reply by ben-sec** [reply]
Yes, on the manager.

**Reply by dougburks** [reply]
OK, seems like you're running 2.3.110 with no hotfixes. You could run soup now to install the latest hotfix or you could wait for 2.3.120 which is coming soon. Those errors shouldn't really be an issue.
